how-to/networking/serve-ntp-with-chrony.md

```diff
@@ -1,7 +1,7 @@
 (serve-ntp-with-chrony)=
 # How to serve the Network Time Protocol with Chrony
 
-`timesyncd` and `timedatectl` will generally do the right thing in keeping your time in sync. However, if you also want to serve NTP information then you need an NTP server. 
+As of Ubuntu 25.10 `chrony` is installed by default, functioning as a client and keeping your time in sync. However, if you also want to serve NTP information then you need an NTP server.
```

Comment:
please-add: 
I think this one referring to the client mode could be a link to chrony-client.md - just in case if search engines made it find this one to get there easily.